staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -134,7 +134,7 @@ func (flags *WaitFlags) AddFlags(cmd *cobra.Command) {
 	flags.ResourceBuilderFlags.AddFlags(cmd.Flags())
 
 	cmd.Flags().DurationVar(&flags.Timeout, "timeout", flags.Timeout, "The length of time to wait before giving up.  Zero means check once and don't wait, negative means wait for a week.")
-	cmd.Flags().StringVar(&flags.ForCondition, "for", flags.ForCondition, "The condition to wait on: [delete|condition=condition-name].")
+	cmd.Flags().StringVar(&flags.ForCondition, "for", flags.ForCondition, "The condition to wait on: [delete|condition=condition-name]. The default status value of condition-name is true, you can set false with condition=condition-name=false")
```

Comment:
As we are touching in these bits. Can we add example as well?

"The condition to wait on: [delete|condition=condition-name]. For example, 'kubectl wait --for=condition=ready pod/mypodName --timeout=30s'. This command will wait the pod "mypodName" get ready for 30 seconds. The default status value of condition-name is true, you can set false with condition=condition-name=false"

Reply:
sure.